src/components/cluster/Cluster.tsx

```diff
@@ -375,11 +384,11 @@ function Environment({ environment_name, namespace, id, cluster_id, handleClose,
         {
             environment_name: {
                 required: true,
-                validator: { error: 'This is required field(max 16 chars).', regex: /^.{1,16}$/ }
+                validator: { error: `Allowed: 5-25 chars 'a-z' '0-9' '.'  '-' Start/end with alphanumeric`, regex: /^[a-z](-?\.?[a-z0-9]){4,25}$/ }
             },
             namespace: {
                 required: isNamespaceMandatory,
-                validator: { error: '^[a-z]+[a-z0-9\-\?]*[a-z0-9]+$ pattern should satisfy.', regex: /^[a-z]+[a-z0-9\-\?]*[a-z0-9]+$/ }
+                validator: { error: `Allowed: 5-25 chars 'a-z' '0-9' '.'  '-' Start/end with alphanumeric`, regex: /^[a-z](-?\.?[a-z0-9]){4,25}$/ }
```

Comment:
message is wrong, as per regex, it can start with alphabet only but can end in alphabet and number